vuepress/docs/next/tutorials/devops/tls-with-cert-mgr.md

```diff
@@ -16,9 +16,9 @@ kubectl create namespace cert-manager
 
 2. Follow the install instructions to create the `cert-manager` resources in this namespace. The install can be as simple as the following command if the default configuration is acceptable: 
 ``` bash
-kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.12.0/cert-manager.yaml -n cert-manager
+kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.14.0/cert-manager.yaml -n cert-manager
 ```
-
+<!-- Should this be changed to 1.14??? v1.12.0 requires app.kubernetes.io/version: "v1.12.0” while v1.14.0 requires v1.14.0 -->
```

Comment:
@pietrangelo can you please add something to this point?

Reply:
@firegloves just confirmed with Pietrangelo via Slack that this doesn't need to be updated